webapp/src/plugin.jsx

```diff
@@ -207,7 +207,7 @@ export default class DemoPlugin {
 
         registry.registerTranslations(getTranslations);
 
-        registry.registerNeedsTeamRoute('/teamtest', () => 'Demo plugin team route.');
+        registry.registerNeedsTeamRoute('/teamtest', () => <RouterShowcase/>);
```

Comment:
I think we can directly use the component here, it's a bit more readable:

```suggestion
        registry.registerNeedsTeamRoute('/teamtest', RouterShowcase);
```